core/src/main/java/rdx/works/core/preferences/PreferencesManager.kt

```diff
@@ -120,6 +125,11 @@ class PreferencesManagerImpl @Inject constructor(
             preferences[KEY_DEVICE_ROOTED_DIALOG_SHOWN] ?: false
         }
 
+    override val lastSyncedAccountsWithCE: Flow<String?> = dataStore.data
```

Comment:
Maybe replace `String?` with `Hash?` like the rest of the methods here to be more safe on what we are dealing with?

Reply:
Also why do we need to mark it as optional?

Reply:
Hmm, while it's good to know the exact data type, I think it's better to keep the PreferencesManager responsible only for storing and retrieving the data, with no processing or any logic around it, that's why it's an optional String. Let me know your thoughts on this.

Reply:
I agree with Michael on this, and I also agree on not having logic in PreferencesManager, but I think encoding/decoding a String is not so bad if it's there. 
Not a hard requirement from my side though.

Reply:
From my point of view, having encoding/decoding here breaks SRP and I agree that it's quite hard to always respect it, but here it's quite easy to do so. I'd leave it as is.